package.json

```diff
@@ -56,10 +56,11 @@
   },
   "dependencies": {
     "@polymer/polymer": "^3.0.0",
-    "@vaadin/vaadin-themable-mixin": "^1.6.1",
+    "@vaadin/vaadin-avatar": "^1.0.3",
```

Comment:
Consider using `v2.0.0-alpha1` which will be soon stable and part of Vaadin 19.

Reply:
Done.